docs/topics/cli-run.adoc

```diff
@@ -25,56 +27,66 @@ $ ./mta-cli --input /path/to/jee-example-app-1.0.0.ear --output /path/to/output
 
 . Access the report.
 
-[discrete]
 [id="command-examples_{context}"]
-== {ProductShortName} Command examples
+== {ProductShortName} command examples
 
 [discrete]
 === Running {ProductShortName} on an application archive
 
 The following command analyzes the `com.acme` and `org.apache` packages of the link:https://github.com/windup/windup/blob/master/test-files/jee-example-app-1.0.0.ear[jee-example-app-1.0.0.ear] example EAR archive for migrating from JBoss EAP 5 to JBoss EAP 7:
```

Comment:
I know this is out of scope but we are not allowed to link to GitHub in downstream docs without TAM approval.